.github/workflows/CI.yml

```diff
@@ -1,11 +1,12 @@
 name: CI
 
 # Controls when the action will run. Triggers the workflow on push or pull request
-# events but only for the main branch
+# events but only for the main and next branches
```

Comment:
There is no next branch here 😬  (which is reflected in more places below)

Reply:
I know, but I wouldn't bother with that. In the future we might add a `next` branch and the project is then already prepared for it. What do you think?

Reply:
Feels like premature optimisation - it's not our reality 🤔 

But it's fine, it works anyway 🙃